desktop/models/user.coffee

```diff
@@ -59,8 +60,16 @@ module.exports = class User extends Backbone.Model
     CurrentUser = require './current_user.coffee'
     LoggedOutUser = require './logged_out_user.coffee'
 
-    CurrentUser.orNull() or
-    new LoggedOutUser attributes
+    user = CurrentUser.orNull() or new LoggedOutUser attributes
+
+    if user instanceof CurrentUser
+      RavenClient.setUserContext
```

Comment:
Ah, noted. I'll go ahead and remove this just to keep the boundary secure. Until we start refining what error data we collect I don't think this is that relevant.